src/app/payment-hub/request-to-pay/helper/ams-short-codes.ts

```diff
@@ -0,0 +1,34 @@
+export const amsShortCodes = [
+    {
+        option: 'ERPLY_TILL',
+        value: '9499081'
```

Comment:
@simiyu17 suggesting using environment variables for AMS short code values so that when there's a change we don't have to modify the codebase, we just change the env variables